src/components/vm/consoles/VncConsole.tsx

```diff
@@ -53,27 +63,44 @@ export const VncConsole = ({
     shared = false,
     credentials,
     repeaterID = '',
-    vncLogging = 'warn',
     consoleContainerId,
-    onConnected = (element) => {},
+    onConnected = () => {},
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2267-42a6eadf-20250808-101729-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)